pkg/runtimeconfig/runtimeconfig.go

```diff
@@ -27,7 +27,23 @@ func Get() *ecv1beta1.RuntimeConfigSpec {
 }
 
 func Cleanup() {
-	os.RemoveAll(EmbeddedClusterTmpSubDir())
+	emptyTmpDir()
```

Comment:
we already have `helpers.RemoveAll`. can you please use that instead? it also has tests